devsetup/edpm/edpm-play.yaml

```diff
@@ -1,242 +0,0 @@
-                  edpm_nova_compute_neutron_project_domain_name: Default
-                  edpm_nova_compute_neutron_project_name: service
-                  edpm_nova_compute_neutron_region_name: regionOne
-                  edpm_nova_compute_neutron_user_domain_name: Default
-                  edpm_nova_compute_neutron_username: neutron
-                  edpm_nova_compute_oslo_messaging_notifications_transport_url: '_EDPM_NOVA_NOTIFICATIONS_TRANSPORT_URL_'
-                  edpm_nova_compute_placement_auth_type: password
-                  edpm_nova_compute_placement_auth_url: http://keystone-internal.openstack.svc:5000
-                  edpm_nova_compute_placement_password: 12345678
-                  edpm_nova_compute_placement_project_domain_name: Default
-                  edpm_nova_compute_placement_project_name: service
-                  edpm_nova_compute_placement_region_name: regionOne
-                  edpm_nova_compute_placement_user_domain_name: Default
-                  edpm_nova_compute_placement_username: placement
-                  edpm_nova_compute_placement_valid_interfaces: internal
-                  edpm_nova_compute_service_user_auth_type: password
-                  edpm_nova_compute_service_user_auth_url: http://keystone-internal.openstack.svc:5000
-                  edpm_nova_compute_service_user_password: 12345678
-                  edpm_nova_compute_service_user_project_domain_name: Default
-                  edpm_nova_compute_service_user_project_name: service
-                  edpm_nova_compute_service_user_region_name: regionOne
-                  edpm_nova_compute_service_user_send_service_user_token: true
-                  edpm_nova_compute_service_user_user_domain_name: Default
-                  edpm_nova_compute_service_user_username: nova
-                  edpm_nova_compute_vnc_novncproxy_base_url: http://192.168.24.3:6080
-                  edpm_ovn_dbs:
-                  - _EDPM_OVN_DBS_
-            vars:
-                registry_name: "quay.io"
-                registry_namespace: "tripleozedcentos9"
-                image_tag: "current-tripleo"
-                edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
-                edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
-                edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
-                edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
-                edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
-                edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
```

Comment:
In Zuul CI. We push containers to different registries. The same is going to be true for downstream.
We need to specify these urls in dataplane CRD so that we can kustomize it. I am going to propose a PR in dataplane operator to move it there.

Reply:
Here is the pull request for the same: https://github.com/openstack-k8s-operators/dataplane-operator/pull/110